cloudferrylib/os/actions/check_cloud.py

```diff
@@ -12,13 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import time
```

Comment:
I would also add space after it, like specified in that doc ;)
But it's super-minor of course.